### ISSUE TITLE
cublas verison

### ISSUE THREAD

**Issue by Yfeng-44** [tekdogan/gcn#2]
Hi,

It's a very interesting work. Thanks a lot for doing this!

Can I know what is the cublas version in this work (and probably the CUDA version as well)

Thanks,
Yuan

**Comment by tekdogan** [comment]
Hi @Yfeng-44 ,

Thank you for your interest in our work!

The cuBLAS and CUDA versions of this project are 10.2.

Best,
Taha

**Comment by Yfeng-44** [comment]
Hi @tekdogan ,

Thanks for replying. Could I know do you use CUDA 10.2 (and cublas) for GPGPUSim as well? in the log, there's 8.0 in the logfile.

Yuan

**Comment by tekdogan** [comment]
Hi @Yfeng-44 ,

Yes, we also used CUDA 10.2 (and cuBLAS) for GPGPUSim as well.

Actual experiments were run on a V100 with version 10.2. However, we also made earlier tests on Pascal, where we used version 8.0. Logs are probably heritage from the previous machine.

Best,
Taha

**Comment by Yfeng-44** [comment]
Hi @tekdogan ,

Thanks for your information, but I got cublass error when I run SAG with PubMed datasets on a fresh clone with the same setting (V100 GPU, CUDA  cuBLAS 10.1 and 10.2). It yields either CUBLAS_STATUS_NOT_INITIALIZED or CUBLAS_STATUS_EXECUTION_FAILED. 

I've tried more than one machines. Is there anything I miss? (this maybe unrelated to the topic of this thread, raise another issue)

Thanks.
Yuan
